src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -27,6 +26,18 @@
 
 namespace OpenDebugAD7
 {
+    // not in the DAP dll yet
```

Comment:
Looks like we need to enable this to be generated, it was not created because no one was using this event.

Reply:
You can leave this in for now. When theres an update to the protocol, I will remove and update the package.